BlueprintUICommonControls/Sources/AccessibilityFocus.swift

```diff
@@ -0,0 +1,117 @@
+import BlueprintUI
+import UIKit
+
+/// Enables VoiceOver focus to jump to the wrapped element
+/// either on appearance or via a trigger that can be manually fired.
+public struct AccessibilityFocus: Element {
+
+    /// The element that will have the focus.
+    public var wrapping: Element
+
+    /// The type of accessibility notification that will be triggered.
+    public var notificationType: UIAccessibility.Notification
+
+    /// A object that can be held on to by the caller to manually trigger a focus.
+    public var trigger: Trigger
+
+    // MARK: Element
+
+    public var content: ElementContent {
+        ElementContent(child: wrapping)
+    }
+
+    public func backingViewDescription(with context: ViewDescriptionContext) -> ViewDescription? {
+        View.describe { config in
+            config[\.isAccessibilityElement] = false
+            config.apply { view in
+                view.apply(model: self)
+            }
+        }
+    }
+}
+
+// MARK: Backing view
+
+extension AccessibilityFocus {
+    private final class View: UIView {
+
+        func apply(model: AccessibilityFocus) {
+            model.trigger.action = { [weak self] in
```

Comment:
The pattern I'd recommend is to hold a reference to the trigger on `View`, so that you can clear out old triggers when setting this, e.g.:
```swift
var accessibilityFocusTrigger: Trigger {
  didSet {
    oldValue?.action = nil
    accessibilityFocusTrigger.action = { [weak self] in
      // action to perform on trigger
    }
  }
}
```